lib/modules/apostrophe-i18n-templates/index.js

```diff
@@ -77,7 +76,17 @@ module.exports = {
 
           self.apos.modules['apostrophe-i18n-static'].insert(req, piece, { permissions: false }, (err, pieceAdded) => {
             if (err) {
-              console.error('Error while inserting apostrophe-i18n-static piece', err.message);
+              if (err.toString().match(/E11000/)) {
+                // The database has it but the flat file does not. We still need
+                // to trigger a regeneration of the flat file
+                return self.apos.modules['apostrophe-i18n-static'].updateGeneration(req, piece, function(err) {
```

Comment:
Question here: why do we launch updateGeneration here where we had a duplicate key error - so no new piece inserted?